app/(auth)/sign-up-flow/birthday.tsx

```
@@ -23,29 +21,42 @@ export default function BirthdayScreen() {
     }
   };
 
-  const validateAge = (dateString: string) => {
-    // Split the date string into components
+
+const validateAge = (dateString: string) => {
     const [month, day, year] = dateString.split('/').map(num => parseInt(num, 10));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)